recipes-bsp/imx-test/imx-test_git.bb

```diff
@@ -6,7 +6,7 @@ SUMMARY = "Test programs for i.MX BSP"
 DESCRIPTION = "Unit tests for the i.MX BSP"
 SECTION = "base"
 LICENSE = "GPLv2"
```

Comment:
Here it must be adjusted as well.

Reply:
Correct, missed this part. I've adjusted it to `GPLv2+` in the force-push.

Reply:
I think what Otavio meant is that 'GPLv2' and friends have also been deprecated.
So the line should be changed to "LICENSE = "GPL-2.0-only"  (or GPL-2.0-or-later).

https://spdx.github.io/spdx-spec/appendix-I-SPDX-license-list/

Reply:
Hello @MaxKrummenacher 

> 
> 
> I think what Otavio meant is that 'GPLv2' and friends have also been deprecated.

This I've also assumed, but haven't seen. According to [meta/conf/licenses.conf](https://git.openembedded.org/openembedded-core/tree/meta/conf/licenses.conf#n34) `GPLv2+` is still present in _SPDXLICENSEMAP_.

> So the line should be changed to "LICENSE = "GPL-2.0-only" (or GPL-2.0-or-later).
> 
> https://spdx.github.io/spdx-spec/appendix-I-SPDX-license-list/

So far, I haven't seen any occurrence of `LICENSE = "GPL-2.0-or-later"` in OE-Core, recipes are rather using the `LICENSE = "GPLv2+"` syntax.

`GPL-2.0-or-later`  on the other hand is used only in `LIC_FILES_CHKSUM` in several OE-Core recipes, e.g. [meta/recipes-core/util-linux/util-linux_2.36.2.bb](https://git.openembedded.org/openembedded-core/tree/meta/recipes-core/util-linux/util-linux_2.36.2.bb).

In this regard, I was just following the OE-Core approach of not changing the `LICENSE = "GPLv2+"` to `LICENSE = "GPL-2.0-or-later"`.

Please let me know if that makes sense and clarify things?

@otavio Should we change the `LICENSE` to full _SPDX_ identifiers, or stay with current ones (like in OE-Core)?

-- andrey

Reply:
I prefer `GPLv2+`. The checksum will need to match the files and this out of recipe control so we need to use whatever the package use. If eventually OE-Core moves to `GPL-2.0-or-later` in the future we can replace it all at once when needed.

Reply:
I'm fine with that.